source/pymongo-async-migration.txt

```diff
@@ -0,0 +1,339 @@
+
+          count = await collection.estimated_document_count()
+
+   * - ``count_documents()``
+     - .. code-block:: python
+
+          count = await collection.count_documents(...)
+
+   * - ``create_index()``
+     - .. code-block:: python
+          
+          await collection.create_index(...)
+
+   * - ``create_indexes()``
+     - .. code-block:: python
+          
+          await collection.create_indexes(...)
+
+   * - ``drop_index()``
+     - .. code-block:: python
+
+          await collection.drop_index(...)
+
+   * - ``drop_indexes()``
+     - .. code-block:: python
+
+          await collection.drop_indexes()
+
+   * - ``list_indexes()``
+     - .. code-block:: python
+
+          indexes = await collection.list_indexes()
+
+   * - ``index_information()``
+     - .. code-block:: python
+
+          info = await collection.index_information()
+
+   * - ``list_search_indexes()``
+     - .. code-block:: python
+
+          indexes = await collection.list_search_indexes()
+
+   * - ``create_search_index()``
+     - .. code-block:: python
+
+          await collection.create_search_index(...)
+
+   * - ``create_search_indexes()``
+     - .. code-block:: python
+
+          await collection.create_search_indexes(...)
+
+   * - ``drop_search_index()``
+     - .. code-block:: python
+
+          await collection.drop_search_index(...)
+
+   * - ``update_search_index()``
+     - .. code-block:: python
+
+          await collection.update_search_index(...)
+   
+   * - ``options()``
+     - .. code-block:: python
+
+          options = await collection.options()
+   
+   * - ``aggregate()``
+     - .. code-block:: python
+
+          async for doc in await collection.aggregate(...):
+          ...
+
+   * - ``aggregate_raw_batches()``
+     - .. code-block:: python
+
+          async for batch in await collection.aggregate_raw_batches(...):
+          ...
+
+   * - ``rename()``
+     - .. code-block:: python
+
+          await collection.rename(...)
+
+   * - ``distinct()``
+     - .. code-block:: python
+
+          distinct_values = await collection.distinct(...)
+
+   * - ``find_one_and_delete()``
+     - .. code-block:: python
+
+          result = await collection.find_one_and_delete(...)
+
+   * - ``find_one_and_replace()``
+     - .. code-block:: python
+
+          result = await collection.find_one_and_replace(...)
+
+   * - ``find_one_and_update()``
+     - .. code-block:: python
+
+          result = await collection.find_one_and_update(...)
+
+Migrate From Motor
+------------------
+
+The {+driver-async+} driver behaves nearly identically to the Motor library. In
+most cases, you can directly migrate existing Motor applications to
+{+driver-short+}, changing only the application's import statements, and
+by using ``AsyncMongoClient`` in place of ``MotorClient``.
+
+The following section shows the method signature changes that you must make when
+migrating from Motor to the {+driver-async+} driver:
```

Comment:
```suggestion
The {+driver-async+} driver behaves nearly identically to the Motor library. In
most cases, you can directly migrate existing Motor applications to
{+driver-short+}, changing only the application's import statements.
The following code shows the difference in imports to use an XXX in Motor compared to Pymongo async:

.. code-block:: python
   
   # Motor Imports for zzz
   xxxx
   
   # PyMongo Async imports for zzz
   xxxx
   
You must also use the xxx type to create a client instead of MotorClient

The following section describes the method signature changes that you must make when
migrating from Motor to the {+driver-async+} driver.
```